app/cafe.py

```diff
@@ -0,0 +1,28 @@
+from datetime import date
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("You cannot enter without a vaccine")
+        vaccine_info = visitor["vaccine"]
```

Comment:
you can simplify lines 17-22 to one if condition and raise only one  OutdatedVaccineError

Reply:
Did you change this?